### PR TITLE
GitHub actions -- Linting

### PR DESCRIPTION
**Pull request by ethanolmethanol**
yes

### CODE REVIEW

**Comment by ethanolmethanol** [comment]
My precious commits shall never be forgotten. Even if they disappeared on this PR.

File: front/.vscode/extensions.json
Comment (opaleboreale) [inline front/.vscode/extensions.json:1]: a supprimer

File: front/.vscode/launch.json
Comment (opaleboreale) [inline front/.vscode/launch.json:1]: a supprimer

File: front/.vscode/tasks.json
Comment (opaleboreale) [inline front/.vscode/tasks.json:1]: a supprimer

**Review by opaleboreale — CHANGES_REQUESTED** [review]
ok

File: front/src/app/components/error-message/error-message.component.css [hunk 1]
```diff
@@ -1,11 +1,12 @@
-@keyframes fadeIn {
+@keyframes faade-in {
```
Comment (ethanolmethanol) [inline front/src/app/components/error-message/error-message.component.css:1]: "faade-in"
What's next? faaaaaaade-in? Emphasis on the slowness of the fading?

File: front/src/app/components/status-icon/status-icon.component.css [hunk 2]
```diff
@@ -25,21 +25,22 @@
   border: 1px solid #f5c6cb;
   border-radius: 5px;
   padding: 5px 10px;
-  box-shadow: 0 2px 4px rgba(0,0,0,0.1);
+  box-shadow: 0 2px 4px rgb(0 0 0 / 10%);
   white-space: nowrap; /* Prevents the text from wrapping */
   transition: opacity 0.3s ease-in-out; /* Smooth transition */
   opacity: 0; /* set initial opacity to 0 */
-  animation: fadeIn 0.9s ease-in-out forwards; /* Apply the fadeIn animation */
+  animation: faade-in 0.9s ease-in-out forwards; /* Apply the faade-in animation */
```
Comment (ethanolmethanol) [inline front/src/app/components/status-icon/status-icon.component.css:32]: faaaaaaaade

File: front/src/app/pages/sign-in-page/sign-in/sign-in.component.css [hunk 3]
```diff
@@ -34,31 +34,33 @@ form-group {
   font-size: 0.6em;
   border-radius: 10px;
   transition: background-color 0.3s ease;
-  animation: fadeIn 1s ease-out;
+  animation: faade-in 1s ease-out;
```
Comment (ethanolmethanol) [inline front/src/app/pages/sign-in-page/sign-in/sign-in.component.css:37]: faaaaaaaaaaaaaaaaaaaade

File: front/src/app/pages/sign-up-page/sign-up/sign-up.component.css [hunk 4]
```diff
@@ -52,21 +52,22 @@
     font-size: 0.6em;
     border-radius: 10px;
     transition: background-color 0.3s ease;
-    animation: fadeIn 1s ease-out;
+    animation: faade-in 1s ease-out;
```
Comment (ethanolmethanol) [inline front/src/app/pages/sign-up-page/sign-up/sign-up.component.css:55]: faaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaade

**Review by ethanolmethanol — COMMENTED** [review]
Good job copy-pasting my commits! Faaaaaaaaaaaaaaaaaaaaade

**Review by opaleboreale — APPROVED** [review]
all good

**Review by romann-broque — APPROVED** [review]
Let me introduce you... Me!

**Pull request merged**